exporter/awsemfexporter/emf_exporter_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
 	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
 	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
+	internaldata "github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus"
```

Comment:
This is not following the import grouping rules used by this repo and will fail the lint checks.  Revert the changes to this file.

Reply:
Done too !! Thanks for your insight !!!